sandboxnu/mfa-form-automator
# 202 feature vercel blob storage setup

## Description
- Implement vercel blob storage
- Refactor code to upload files to blob storage and pull from storage to view in embeds and react-pdf.
- Refactor `FormImageCard` to contain seperate Document viewer
- Fix packages and config files 😵‍💫 



## Motivation and Context


- Allow forms to be uploaded when creating a form template, and pulling that form when creating a form instance
- Clean up some messy code and package management regarding react-pdf

Closes [#202]

## How has this been tested?




- Upload form template
- Verify that `formDocLink` in backend for created points to form in vercel blob
- Create form instance using form template

## Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/051e2f65-4b26-4f05-a698-a0993251e547)
![image](https://github.com/user-attachments/assets/7cd4b587-e71d-4f2b-8410-84dd62c6ffd8)
![image](https://github.com/user-attachments/assets/a704899a-b6df-4157-b36e-5ef5b8adf07a)
![image](https://github.com/user-attachments/assets/b658aa03-a31d-4b1f-8c7a-ffba069bec31)


## Types of changes




## Checklist:
